src/connection.js

```diff
@@ -904,6 +916,23 @@ export class Connection {
     return res.result;
   }
 
+  /**
+   * Fetch a list of Transactions and transaction statuses from the cluster
+   */
+  async getBlock(
+    slot: number,
+  ): Promise<Array<[Transaction, SignatureSuccess | TransactionError | null]>> {
```

Comment:
esdoc is complaining about this type ^

It looks like it's supposed to be: `[Transaction, SignatureSuccess] | ...` just missing a closing bracket

Reply:
The closing bracket is after the `null`...
I tried putting the whole OR expression (the second value in the tuple) in parens, but prettier removed them

Reply:
Maybe I have to be explicit?
`[Transaction, SignatureSuccess] | [Transaction, TransactionError] ...`

Reply:
Yes, the explicit tuples approach works.
Thanks, @jstarry !